query/src/test/scala/filodb/query/exec/SlidingWindowIteratorSpec.scala

```diff
@@ -133,7 +133,8 @@ class SlidingWindowIteratorSpec extends FunSpec with Matchers {
                     1538416644000L->210170299d,
                     1538416649000L->210172635d)
 
-  it ("should ignore out of order samples for RateFunction") {
+  // Out of order samples are now handled by ingestion
+  ignore("should ignore out of order samples for RateFunction") {
```

Comment:
FYI.. This test accepts out of order data above.

Reply:
Right, but the iterator for dropping out of order samples is disabled now, so the test doesn't make sense anymore.  Right?

Unless one were to redo all the query tests to be actually based on raw TimeSeriesPartitions, which is probably a good idea IMO.

Reply:
Reverted.